pkg/arvo/sys/hoon.hoon

```diff
@@ -6782,6 +6782,7 @@
   ::                                            ::::::  special
     {$zpcm p/hoon q/hoon}                               ::  !,
     {$zpbn p/hoon}                                      ::  !>
+    {$zpgl p/spec q/hoon}                               ::  !<
```

Comment:
This naming is inconsistent with the rest of hoon.hoon wrt pronounceables (ban/led vs gal/gar)

Reply:
you're right, I'll change to zpld.

Reply:
we want gal/gar not led/ban

Reply:
Yes, but we require consistency. The AST nodes should be migrated all at once; it would be confusing for them to simultaneously use two incompatible schemes